README.md

```diff
@@ -410,19 +374,22 @@ go run cmd/tools/cmd/main.go claim-add --claimfile=claim.json --reportdir=/home/
 When run the CNF test suite will output a report to the terminal that is primarily useful for Developers to evaluate and
 address problems.  This output is similar to many testing tools.
 
-Here's an example of a Test pass.  It shows the Test running a command to extract the contents of `/etc/redhat-release`
-and using a regular expression to match allowed strings.  It also prints out the string that matched.:
+#### Test successful output example
+Here's an example of a Test pass.  It verifies that the CNF is using a replica set:
 
 ```shell
 ------------------------------
-generic when test(test) is checked for Red Hat version 
-  Should report a proper Red Hat version
-  /Users/$USER/cnf-cert/test-network-function/test-network-function/generic/suite.go:149
-2020/12/15 15:27:49 Sent: "if [ -e /etc/redhat-release ]; then cat /etc/redhat-release; else echo \"Unknown Base Image\"; fi\n"
-2020/12/15 15:27:49 Match for RE: "(?m)Red Hat Enterprise Linux Server release (\\d+\\.\\d+) \\(\\w+\\)" found: ["Red Hat Enterprise Linux Server release 7.9 (Maipo)" "7.9"] Buffer: "Red Hat Enterprise Linux Server release 7.9 (Maipo)\n"
+lifecycle when Testing owners of CNF pod 
+  Should be only ReplicaSet
+  /home/deliedit/redhat/github/david/ssh/test-network-function/test-network-function/lifecycle/suite.go:339
```

Comment:
some stuff in here should be removed.  I think a merge went sideways :)

Reply:
This is an example of output similar to what was there before, I just updated with new suites